domains/caeden.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "criticalswateam1",
+           "email": "",
+           "discord": "832663333529845772",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Apvo6s4H-N0IYTP9lXRWxDnger4mKz8SwJGP8ZLAfF3huZzg6x6MVWgllYDVQE9YAK9P2ejMe2u3hJi_tN4F7finYiq5-UGwn2jx1nZVUEcY5Vp8Cv8qX2gknKhj_tCzTE9tDLnWkoojaR03fvtFe92fcD6A3WapGxlS7v3MMSwnyiD8CcsEETtyGZ8c6gmzsy7XR3m2Z8lH9wK-N5s4Bd3J0XLp6K9C4L5sr7uJAk_jTwFzVyrEL8cFASWir59v1Q6lqoTkpz-f39AUc07InHy1_wRI4TZoMD2pPfRl4QHJiw2w_ErPGxNDOnrc68ppp7tp8TGx-umicnjnLramHw.uYEuEkBOo4KlBmzGBYm98Q.S0aVi32yMnRDx2lFoYvMeGJQSS4FSV-65I5o4fKo_pkZkmWt6z6JAOtHD2ui6aZ13xq5qHRPSBP6PQw7TRhoAKWE3xhVfYoWBhLXPvqeS3ZXSfZOFfm2RR98uDNh9mSQsnZqe9TRcPpPRR1SGXJe_A.26YtzGbeWiDssFFjQx0d2A"
+        },
+    
+        "record": {
+            "CNAME": "https://caeden-6crdt2526-criticalswateam1s-projects.vercel.app/"
```

```suggestion
            "CNAME": "caeden-6crdt2526-criticalswateam1s-projects.vercel.app"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).